### DISCUSSION TITLE
Toggling "automatically download beatmaps" when spectating redownloads beatmap if already downloaded

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16824] [Q&A] [answered]
I don't think a user would be spamming it, but the use case that i can think of is if you start spectating someone and want to temporarily turn the feature on, it would still trigger a redownload of the map.

https://user-images.githubusercontent.com/53081750/152952383-69e270ae-2c5a-4835-9fc3-4e60dee22e65.mp4

[runtime.log](https://github.com/ppy/osu/files/8022228/runtime.log)
[database.log](https://github.com/ppy/osu/files/8022226/database.log)
[network.log](https://github.com/ppy/osu/files/8022227/network.log)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16825
